assignments/003-illumina-sequencing-questions/illumina-seq-homework.md

```diff
@@ -0,0 +1,58 @@
+
+
+These are the questions that appear in the video about Illumina sequencing by synthesis that
+can be viewed at: [https://eriqande.github.io/erics-captioned-vids/vids/illumina-sbs/](https://eriqande.github.io/erics-captioned-vids/vids/illumina-sbs/).
+
+Copy this template to a new file in the same directory called simply
+`illumina-seq-homework.md`.  Please answer each question by typing in the space below each question, or by retaining the words corresponding to
+your answers in the "dual-choice in parentheses" questions.
+
+When you are done.  Save the file then commit it
+to a branch called `illumina-seq` and push it back to your
+fork of the course repo on GitHub then send me a pull request.
+
+
+
+* Q1: True or False. This animation is showing that each of the two "fragments" is a double-stranded segment of DNA. (_from 30.0 to 33.0 seconds in the video_).
+
+False - each fragment is one strand
+
+* Q2: How does the sequence of the fragment on top compare to the sequence of the fragment on the bottom? (_from 36.5 to 39.0 seconds in the video_).
+
+They are complementary
+
+* Q3: What is the purpose of the sequencing binding sites? Your answer should include the words "oligonucleotide," "polymerase," "3'-hydroxyl," and "synthesize." (_from 39.0 to 40.5 seconds in the video_).
+
+The sequencing binding sites allow oligonucleotide primers to attach to the DNA strand. The 3'-hydroxyl on the primer is a starting point for new strand synthesis (by a polymerase) that is utilized by Illumina.
+
+* Q4: Given the direction of travel of the polymerase, is the Original Template anchored to the lawn by its 3' or its 5' end? (_from 78.0 to 81.0 seconds in the video_).
+
+They move 5' to 3'. It is anchored by the 5' end.
+
+* Q5: How does the remaining strand relate to the Original Template? Is the remaining strand anchored by its 3' or 5' end? (_from 85.0 to 88.0 seconds in the video_).
+
+It is the reverse complement. This will be anchored by the 5' end.
+
+* Q6: Choose one from the parentheses: the purple part on the remaining strand is (the same as || complementary to) the purple lawn oligos.  (_from 88.2 to 92.2 seconds in the video_).
+
+The same as
+
+* Q7: Choose one from each set of parentheses: the blue part of the remaining strand is at the (3'|5') end and is (the same as|complementary to) the blue oligos on the lawn.  (_from 93.0 to 96.0 seconds in the video_).
+
+The same as
```

Comment:
The remaining strand is attached by the 5' end, so the blue part is at the 3' end, and will stick to the blue ones on the lawn so I think it is complementary.